src/backend/drm/surface/gbm.rs

```diff
@@ -430,6 +446,33 @@ impl Drop for FbHandle {
     }
 }
 
+struct BufferWrapper<'a, B>(&'a B);
```

Comment:
Do we need two of these? Having a duplicate type is begging for behavior to desync.

Reply:
There aren't two, later commits do extract and unify the code.